airflow/api/auth/backend/kerberos_auth.py

```diff
@@ -64,16 +63,19 @@
 log = logging.getLogger(__name__)
 
 
-CLIENT_AUTH: tuple[str, str] | Any | None = HTTPKerberosAuth(service="airflow")
-
-
```

Comment:
I can't find any usage of `CLIENT_AUTH` outside of the tests. Maybe it intends to be used into the user code, so I move it under the `__getattr__`